Assets/Mirror/Examples/Basic/Scripts/Player.cs

```diff
@@ -13,11 +13,12 @@ public class Player : NetworkBehaviour
         public Text playerNameText;
         public Text playerDataText;
 
-        // These are set in OnStartServer and used in OnStartClient
+        // These are set in BasicNetManager::OnServerAddPlayer
+        [Header("SyncVars")]
         [SyncVar]
-        int playerNo;
+        public int playerNo;
```

Comment:
playerNumber would be a better name, unless this is supposed to mean “player no” literally

Reply:
Done